src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -610,6 +645,18 @@ protected override void HandleInitializeRequestAsync(IRequestResponder<Initializ
                 });
             }
 
+            List<ColumnDescriptor> additionalModuleColumns = null;
+            if (responder.Arguments.ClientID == "visualstudio")
+            {
+                additionalModuleColumns = new List<ColumnDescriptor>();
+                additionalModuleColumns.Add(new ColumnDescriptor("vsLoadAddress", "Load Address", "string", ColumnDescriptor.TypeValue.String));
```

Comment:
The builds are outputting warnings for ColumnDescriptor
```
##[warning]src\OpenDebugAD7\AD7DebugSession.cs(652,45): Warning CS0618: 'ColumnDescriptor.ColumnDescriptor(string, string, string, ColumnDescriptor.TypeValue?, int?)' is obsolete: 'This constructor contains optional parameters and is no longer supported.  Please use the constructor which includes only required parameters.'
```
Please address the other warnings too:
https://dev.azure.com/ms/MIEngine/_build/results?buildId=115779&view=results

You can ignore
```
Warning MSB3277: Found conflicts between different versions of "Newtonsoft.Json" that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed.
```

Reply:
The ColumnDescriptor warning is unavoidable for Timestamp.

Reply:
The full parameterized constructor is obsolete/depreciated. You will want to use either the default constructor or the one with two required parameters.

I prefer just using the default constructor and providing the parameters.

```
additionalModuleColumns.Add(new ColumnDescriptor()
{
   AttributeName = "vsLoadAddress",
   Label = "Load Address",
   Format = "string",
   Type = ColumnDescriptor.TypeValue.String
});
```